lua/diagram/integrations/markdown.lua

```diff
@@ -35,6 +35,9 @@ M.query_buffer_diagrams = function(bufnr)
   for id, node in matches do
     local key = query.captures[id]
     local value = vim.treesitter.get_node_text(node, bufnr)
+    if node:parent():parent():type() == "block_quote" then
+	value = value:gsub("\n>", "\n"):gsub("^>", "")
+    end
```

Comment:
```suggestion
if node and node:parent() and node:parent():parent() and node:parent():parent():type() == "block_quote" then
    value = value:gsub("\n>", "\n"):gsub("^>", "")
end
```

Reply:
Thank you! I believe because of the way you're finding matches with the ts_query the existence of node:parent() is guaranteed so that check might be unnecessary.